core/server/src/proto/journal/journal.proto

```diff
@@ -12,30 +12,28 @@ import "lineage.proto";
 // next available id: 29
 message JournalEntry {
   optional int64 sequence_number = 1;
-  oneof entry {
-    AddMountPointEntry add_mount_point = 2;
-    BlockContainerIdGeneratorEntry block_container_id_generator = 3;
-    BlockInfoEntry block_info = 4;
-    CompleteFileEntry complete_file = 5;
-    CompletePartitionEntry complete_partition = 21;
-    CompleteStoreEntry complete_store = 22;
-    CreateStoreEntry create_store = 23;
-    DeleteFileEntry delete_file = 6;
-    DeleteLineageEntry delete_lineage = 7;
-    DeleteMountPointEntry delete_mount_point = 8;
-    DeleteStoreEntry delete_store = 25;
-    InodeDirectoryEntry inode_directory = 9;
-    InodeDirectoryIdGeneratorEntry inode_directory_id_generator = 10;
-    InodeFileEntry inode_file = 11;
-    InodeLastModificationTimeEntry inode_last_modification_time = 12;
-    LineageEntry lineage = 13;
-    LineageIdGeneratorEntry lineage_id_generator = 14;
-    MergeStoreEntry merge_store = 26;
-    PersistDirectoryEntry persist_directory = 15;
-    AsyncPersistRequestEntry async_persist_request = 16;
-    ReinitializeFileEntry reinitialize_file = 18;
-    RenameEntry rename = 19;
-    RenameStoreEntry rename_store = 28;
-    SetAttributeEntry set_attribute = 27;
-  }
+  optional AddMountPointEntry add_mount_point = 2;
```

Comment:
can we document here saying this is supposed to be only "one of" the following different entries and can be represented by `oneof` field if we use protobuf 2.6. However, due to library compatibility reason, we stay in protobuf 2.5